lib/assessment.rb

```diff
@@ -20,20 +25,43 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+    attr_reader :age
+    attr_writer :name, :location
```

Comment:
This should really be `attr_accessor`, not `attr_writer`, because we almost never want to make something writeable without also making it readable. That'd be weird, right? Having the right to change the value of something, but not see what that something is? It's such an unusual use case that it makes me wonder why Ruby even included it.